apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/PatientE2EBase.java

```diff
@@ -821,7 +821,8 @@ public void testPatientByLogicalIdWithNoResultsAndPaginationRequestedExpect200()
    * expected for Patient by mbi.
    */
   @Test
-  public void testPatientByIdentifierHasAccessJsonWithExpectedMdcKeys() throws IOException {
+  public void testPatientByIdentifierHasAccessJsonWithExpectedMdcKeys()
+      throws IOException { // problematic one
```

Comment:
I think we can remove this comment too 
```suggestion
      throws IOException {
```